dashboard/app/models/script_level.rb

```diff
@@ -32,10 +33,16 @@ class ScriptLevel < ActiveRecord::Base
 
   belongs_to :script
   belongs_to :lesson, foreign_key: 'stage_id'
+
+  # This field will only be present in scripts which are being edited in the
+  # new script / lesson edit GUI.
+  belongs_to :activity_section
```

Comment:
idea: it could be a good idea to set up a per-script flag indicating that the script can (and can only) be edited using the new edit GUI. then we can enforce via validation that activity_section is not set for "legacy" scripts.

Reply:
This sounds good to me. When do you think we should do this?

Reply:
I added an item for this in the roadmap for right before we launch.